calico_node/startup/startup.go

```diff
@@ -296,16 +304,16 @@ func configureIPsAndSubnets(node *api.Node) bool {
 			node.Spec.BGP.IPv4Address = cidr.String()
 		} else if node.Spec.BGP.IPv4Address == "" {
 			// No IPv4 address is configured, but we always require one, so exit.
-			fatal("Couldn't autodetect a management IPv4 address:")
-			message("  -  provide an IPv4 address by configuring one in the node resource, or")
-			message("  -  provide an IPv4 address using the IP environment, or")
-			message("  -  if auto-detecting, use a different autodetection method.")
+			log.Warn("Couldn't autodetect a management IPv4 address:")
+			log.Warn("  -  provide an IPv4 address by configuring one in the node resource, or")
```

Comment:
same applies here too I guess.  Should this just be a single log with newlines, or restructured so it isn't trying to be a bulleted list?